boot-cw-plus/src/cw_plus.rs

```diff
@@ -0,0 +1,72 @@
+use crate::*;
+use boot_core::*;
+
+/// Wrapper around the CwPlus contracts for use in cw-multi-tests and deployments.
+pub struct CwPlus<Chain: CwEnv> {
+    pub cw1_whitelist: Cw1Whitelist<Chain>,
+    pub cw1_subkeys: Cw1Subkeys<Chain>,
+    pub cw3_fixed_multisig: Cw3FixedMultisig<Chain>,
+    pub cw3_flex_multisig: Cw3FlexMultisig<Chain>,
+    pub cw4_group: Cw4Group<Chain>,
+    pub cw4_stake: Cw4Stake<Chain>,
+    pub cw20_base: Cw20Base<Chain>,
+    pub cw20_ics20: Cw20Ics20<Chain>,
+}
+
+impl<Chain: CwEnv> Deploy<Chain> for CwPlus<Chain> {
+    type Error = BootError;
+    type DeployData = Empty;
+
+    /// Deploy all cw-plus contracts to the given chain.
+    fn deploy_on(chain: Chain, _data: Empty) -> Result<Self, BootError> {
```

Comment:
Perhaps `upload_to` would be more appropriate. `deploy` implies that it is instantiated as well.

Reply:
In this specific case yes, not arguing that. I'm saying that we'd need to change the trait to reflect that in for it to be true in this specific case while for all other users `deploy_on` makes more sense (look at the WynDex integration and our own Abstract implementation of this trait)

Reply:
will add an `upload_to` fn to the trait